am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/AccessManagementServiceIntegrationTest.java

```diff
@@ -43,10 +45,40 @@ public void createQuery_whenCreatingResourceAccess_ResourceAccessAppearsInDataba
         ams.createResourceAccess(createRecord(resourceId, ACCESSOR_ID, EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS));
 
         int count = jdbi.open().createQuery(
-                "select count(1) from access_management where resource_id = ?")
-                .bind(0, resourceId)
-                .mapTo(int.class)
-                .findOnly();
+            "select count(1) from access_management where resource_id = ?")
+            .bind(0, resourceId)
+            .mapTo(int.class)
+            .findOnly();
+
+        assertThat(count).isEqualTo(1);
+    }
+
+    @Test
+    public void revokeResourceAccess_whenRevokingResourceAccess_ResourceAccessRemovedFromDatabase() {
+        ams.createResourceAccess(createRecord(resourceId, ACCESSOR_ID, EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS));
+
+        ams.revokeResourceAccess(removeRecord(resourceId, ACCESSOR_ID));
+
+        int count = jdbi.open().createQuery(
```

Comment:
This is duplicated. Let's avoid duplicates in tests. People often treat unit test code as second class citizen which is wrong. Due to nature of tests they are often more complex than implementation that makes them more difficult to read. For that reason, and because we often have to fix tests broken by refactoring I suggest making tests as simple as possible. Calling method like `countResourcesById` is much easier to understand than these 5 lines.